ansible/roles/base/tasks/main.yml

```diff
@@ -1,32 +1,27 @@
-
 ---
-
     - name: Check NTP service to make sure its running
       service: name=ntp state=started enabled=yes
-      become: yes
 
     - name: Add Felix Krull's Python PPA
       apt_repository:
-        repo: 'ppa:fkrull/deadsnakes'
-        state: present
+       repo: 'ppa:fkrull/deadsnakes'
+       state: present
 
     - name: Update apt cache
       apt: update-cache=yes cache_valid_time=86400
-      become: yes
-      become_method: sudo
 
     - name: Install Python 3.6
       apt:
-        name: python3.6
-        state: latest
+       name: python3.6
+       state: latest
 
     - name: install python-pip
-      apt: name=python-pip
-      become: yes
+      apt: 
+       name: python3-pip
 
-    - name: install python-virtualenv
-      apt: name=python-virtualenv
-      become: yes
```

Comment:
Does this actually work if you remove the become: yes statements?

Also, as I mentioned before, there are still a handful of spacing issues … and now weird capitalization changes, etc… lol. As with the other items, I’ll probably just let them through and fix them on my own just to get you onto the next items.